sda-commons-server-security/src/main/java/org/sdase/commons/server/security/SecurityBundle.java

```diff
@@ -60,7 +68,14 @@ public void run(T configuration, Environment environment) {
 
     environment.getApplicationContext().setErrorHandler(createNewErrorHandler(environment));
     environment.getAdminContext().setErrorHandler(createNewErrorHandler(environment));
-    environment.jersey().register(WebSecurityHeaderFilter.class);
+    if (supportFrontend) {
+      LOG.info(
+          "Content-Security-Policy headers are configured to support frontends. "
+              + "Services that only serve APIs not enable this feature.");
```

Comment:
NIT: "Services ... don't enable this feature."

Reply:
fixed